var/spack/repos/builtin/packages/msgpack-c/package.py

```diff
@@ -14,6 +14,7 @@ class MsgpackC(CMakePackage):
 
     license("BSL-1.0")
 
+    version("6.1.1", sha256="d7b119f292365d41403b41b40c2fefd82ebd81241e3c658fafe0e638fa54604a")
```

Comment:
Breaking change "Change CMake package name of C++ library to msgpack-cxx". Any dependents need testing?

Reply:
I'm going to pull this out into its own PR for further testing.